src/components/v2/chartDeploymentHistory/ChartDeploymentHistory.component.tsx

```diff
@@ -107,11 +106,19 @@ function ChartDeploymentHistory({
             })
     }, [])
 
+    const getDeploymentData = (_selectedDeploymentTabIndex: number, _selectedDeploymentHistoryIndex: number) => {
+        if (_selectedDeploymentTabIndex === 1 || _selectedDeploymentTabIndex === 2) {
+            const version = deploymentHistoryArr[_selectedDeploymentHistoryIndex].version
```

Comment:
we can directly use this version. no need to create new variable

Reply:
Done